docs/rfcs/002-archival_storage/README.md

```diff
@@ -142,18 +142,16 @@ Digitised content will be ingested using Goobi, which should provide the bag lay
 ```
 b22036593/
 |-- data
-|   |-- b22036593.xml      // mets file
-|  [|-- bb22036593_001.xml // multiple manifestations]
-|  [|-- bb22036593_002.xml // multiple manifestations]
+|   |-- b22036593.xml      // mets "anchor" file for multiple manifestation
+|  [|-- bb22036593_001.xml // mets file for vol 1]
```

Comment:
Can you fix my bb typo while you're in here - thank you!

Reply:
done